nebullvm/inference_learners/base.py

```diff
@@ -471,12 +493,15 @@ def _save_file(
         np.save(output_file, prediction)
 
     def get_inputs_example(self):
```

Comment:
can we add a boolean parameter for generating the data randomly (with `create_model_inputs_onnx`)?